language_service/src/test_utils.rs

```diff
@@ -10,12 +10,49 @@ use qsc::{
     incremental::Compiler,
     line_column::{Encoding, Position, Range},
     location::Location,
+    packages::prepare_package_store,
     target::Profile,
     LanguageFeatures, PackageStore, PackageType, SourceMap, Span,
 };
 use qsc_project::{PackageGraphSources, PackageInfo};
 use rustc_hash::FxHashMap;
 
+const FAKE_STDLIB_CONTENTS: &str = r#"
```

Comment:
Having std lib in the name is a little consfusing to me. Is this just some lib, or is it supposed to be an alternative std lib?

Reply:
Yes, it's supposed to replace the stdlib in most language service tests, since it's more minimal (faster to build) and I can easily add any edge case I want to cover.